pi_ldapproxy/proxy.py

```diff
@@ -255,7 +258,18 @@ class ProxyServerFactory(protocol.ServerFactory):
     def __init__(self, config):
         # NOTE: ServerFactory.__init__ does not exist?
         # Read configuration options.
-        self.agent = Agent(reactor)
+        if config['privacyidea']['verify']:
+            if config['privacyidea']['certificate']:
+                certificate = Certificate.loadPEM(FilePath(config['privacyidea']['certificate']).getContent())
+                log.info('Pinning privacyIDEA HTTPS certificate {certificate!r}', certificate=certificate)
```

Comment:
The log entry looks like this:
```
Jul 14 17:15:23 pxproxy twistd[15354]: 2017-07-14T17:15:23+0200 [pi_ldapproxy.proxy#info] Pinning privacyIDEA HTTPS certificate <Certificate Subject=privacyideaserver Issuer=privacyideaserver>
```

... I find this quite nice, but I can also change the log message to contain the filename instead :)